### ISSUE TITLE
Request - length attributes

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8981]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

I think we have at least 4 attributes doing the same things. If so, this just prevents useful queries and should be cleaned merged into one. If these are necessary, the definitions should be beefed up (because they all look the same to me!).

* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#total_length - I think this should probably be retained.
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#standard_length - I think this is a synonym of total length
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#minimum_standard_length - this makes no sense in Arctos, a lot-representing record can have any number of length attributes and it's easy to find the smallest when necessary. Should be merged into total length, a comment can be added if necessary.
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#maximum_standard_length  - this makes no sense in Arctos, a lot-representing record can have any number of length attributes and it's easy to find the largest when necessary.  Should be merged into total length, a comment can be added if necessary.

## Users

```

 guid_prefix |  sum   
-------------+--------
 UAM:Fish    |      1
 UCM:Fish    |      6
 MLZ:Fish    |  33513
 UWYMV:Fish  |      1
 MSB:Fish    | 194337
 MVZ:Fish    |      2
 UAMObs:Fish |      1
 OGL:Genomic |     33
```

 @ebraker
 @mkoo
 @campmlc
 @happiah-madson
 @falco-rk
 @ewommack
 @esdearmon
 @jandreslopez

HELP!

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by jandreslopez** [comment]
Total and standard lengths are different so whatever changes are made, it should be possible to specify what type of length (standard, fork or total) was measured.  They are meant to capture some length characteristic but what is recorded sometimes varies depending on study logistics or fish species.

**Comment by dustymc** [comment]
>different

Elaborate?

>varies

Yes - I think this is details which belong in method (and if this becomes an update I can do that).

**Comment by jandreslopez** [comment]
Total length - tip of snout to distal edge of caudal fin
Fork length - tip of snout to fork on caudal fin
Standard length - tip of snout to last vertebrae (at the 'joint' that supports the caudal fin)

**Comment by dustymc** [comment]
Adding to AWG for in-person discussion, clearly these aren't quite as interchangeable as I first thought but I also don't quite fully-maybe see how they're distinct.

**Comment by zygoballus** [comment]
@mkoo - So what system ended up being implemented?

**Comment by dustymc** [comment]
>what system ended up being implemented?

I'd also like to know, still looks like lots of methods in type from here, if there's some consensus it should be added to documentation.

**Comment by Jegelewicz** [comment]
From AWG Issues Meeting today

- Will follow up with fish people to confirm and clarify what length should be better defined. 
- Still in discussion, and no fish people present at meeting.
